extensions/authentication-jwt/src/__tests__/unit/jwt.service.ts

```diff
@@ -19,9 +19,17 @@ describe('token service', () => {
     id: '1',
     name: 'test',
   };
+  type Setter<T> = (value?: T) => void;
   const TOKEN_SECRET_VALUE = 'myjwts3cr3t';
   const TOKEN_EXPIRES_IN_VALUE = '60';
-  const jwtService = new JWTService(TOKEN_SECRET_VALUE, TOKEN_EXPIRES_IN_VALUE);
+  const setCurrentUser: Setter<UserProfile> = userProfile => {
```

Comment:
Hmm why the setter is needed here?

Reply:
I have removed that, and clean up the code